packages/forklift-console-plugin/src/modules/StorageMaps/components/StorageMapsEmptyState.tsx

```diff
@@ -45,8 +45,8 @@ const EmptyStatePlans: React.FC<{ namespace: string }> = ({ namespace }) => {
           <Flex direction={{ default: 'column' }} alignItems={{ default: 'alignItemsCenter' }}>
             <FlexItem>
               <ForkliftTrans>
-                Migration network maps are used to map network interfaces between source and target
-                virtualization providers, at least one source and one target provider must be
+                Migration storage maps are used to map source data stores to
```

Comment:
Not all provider types use the term data stores for storage.

```suggestion
                Migration storage maps are used to map source storages  to
```